deps/build.jl

```diff
@@ -2,10 +2,45 @@ using BinDeps
 
 @BinDeps.setup
 
+#@BinDeps.if_install begin
+
 libproj = library_dependency("libproj", aliases = ["libproj"])
 
+
 @linux_only begin
-    provides(AptGet, "libproj0", libproj, os = :Linux)
+
+	
+	# AptGet's LibProj is old
+	#provides(AptGet, "libproj0", libproj, os = :Linux)  
+
+	# so download a newer version
+	libsrc = "http://download.osgeo.org/proj"
+	libproj_ver = "4.9.2" # from source http://download.osgeo.org/proj/
```

Comment:
It's nice to be using a specific version, not just whatever the latest release is.